src/state/dimension.js

```diff
@@ -154,6 +153,14 @@ export const getDroppableDimension = ({
       withMargin: getFragment(getWithSpacing(withWindowScroll, margin)),
       withMarginAndPadding: getFragment(getWithSpacing(withWindowScroll, add(margin, padding))),
     },
+    container: {
+      scroll: {
```

Comment:
In some cases the droppable is the scrollable element itself - are we catering for that? And does the language used reflect that?